timeutil/timeutil_test.go

```diff
@@ -6,20 +6,156 @@ import (
 	"time"
 )
 
+func TestStartOfDay(t *testing.T) {
+	cases := []struct {
+		in   time.Time
+		want time.Time
+	}{
+		{mustParse(t, "2016-01-01T00:00:00Z"), mustParse(t, "2016-01-01T00:00:00Z")},
+		{mustParse(t, "2016-01-13T00:00:00Z"), mustParse(t, "2016-01-13T00:00:00Z")},
+		{mustParse(t, "2016-01-01T12:34:56Z"), mustParse(t, "2016-01-01T00:00:00Z")},
+		{mustParse(t, "2016-12-30T23:59:59Z"), mustParse(t, "2016-12-30T00:00:00Z")},
+	}
+
+	for _, c := range cases {
+		got := StartOfDay(c.in)
+		if got != c.want {
+			t.Errorf("StartOfDay(%s) =>\n%s, want %s", c.in, got, c.want)
```

Comment:
Is there a particular reason you're not using subtests with `t.Run()`, or is it just a personal preference?

Reply:
The other tests didn't, I wasn't sure what version of Go this is targeted at

Reply:
Oh, that's just because this has been copied out of Desk or something ages ago. Most of the other code uses subtests.